source/features/clean-conversation-headers.tsx

```diff
@@ -0,0 +1,68 @@
+import select from 'select-dom';
+import {observe} from 'selector-observer';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+function initIssue(): void {
+	observe('.gh-header-meta .flex-auto', {
+		add({childNodes}) {
+			// Removes: octocat opened this issue on 1 Jan [·] 1 comments
+			childNodes[4].textContent = childNodes[4].textContent!.replace('·', '');
+
+			// Removes: [octocat opened this issue on 1 Jan] · 1 comments
+			for (const node of Array.from(childNodes).slice(0, 4)) {
+				node.remove();
```

Comment:
I think it's best to also hide this
```suggestion
				node.hidden = true;
```

Reply:
There is a text node

![image](https://user-images.githubusercontent.com/44045911/89753339-778b5f80-db0a-11ea-9375-2f1632f1efa9.png)